preparation.sh

```diff
@@ -1,16 +1,25 @@
 #!/usr/bin/env bash
+set -e
 
 export REBUILD_GRAPHS=true
-export ors.engine.preparation_mode=true
-export ors.services.routing.mode=preparation
+export ORS_ENGINE_PREPARATION_MODE=true
+export ORS_SERVICES_ROUTING_MODE=preparation
+
+mv files/data.osm.pbf original.pbf
+osmconvert -v original.pbf --parameter-file=/parameter-file -o=files/data.osm.pbf
```

Comment:
question: Is there a reason to keep `original.pbf` around in the layer?

Reply:
It's not really kept, since we copy the graphs to a fresh layer [here](https://github.com/mbta/ors-deploy/pull/11/files/5b7a4ada53913489197d71b2e3ab208f09783a0b#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R15). The main thing is that the two files need to be different, and the end state needs to be the data in `files/data.osm.pbf`.

I could remove it if you think it's cleaner.

Reply:
oooh I see, hmmm, I could go either way. I'm good with leaving it.

Reply:
Oh I see the `rm original.pbf` was added, thanks!